blueprints.yaml

```diff
@@ -33,6 +33,15 @@ form:
       validate:
         min: 0
         max: 10
+    max_depth:
+      type: range
+      label: Maximum Level
+      description: Deepest depth to which to generate the tree.
```

Comment:
The `description` here should match the above request, "Depth at which to stop generating the tree."